src/app/ApplicationTemplate.UWP/Views/Styles/Controls/Button.xaml

```diff
@@ -1,5 +1,171 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:u="using:Nventive.View.Controls">
+
+	<!-- TODO: Replace this style by new Uno.Material MaterialTextButtonStyle with IconPresenter to the right -->
```

Comment:
Is this something that's not available yet?

Reply:
Not that I've seen. Right now material text, contained and outlined button styles have an icon presenter only to the left of the text. So we cannot use ControlExtensions.Icon to set an icon that is on the right of the text.